doc/classes/HTTPRequest.xml

```diff
@@ -173,18 +173,53 @@
 				[b]Note:[/b] Some Web servers may not send a body length. In this case, the value returned will be [code]-1[/code]. If using chunked transfer encoding, the body length will also be [code]-1[/code].
 			</description>
 		</method>
+		<method name="get_dictionary_from_header" qualifiers="const">
```

Comment:
```suggestion
		<method name="get_dictionary_from_headers" qualifiers="const">
```

Why is the `s` missing here? As far as I can see, its in the other places in the code.

Reply:
Thanks for spotting the typo!